modules/snooker/src/main/ts/ephox/snooker/api/Generators.ts

```diff
@@ -166,12 +171,52 @@ const merging = function (generators: Generators) {
       // Remove any width calculations because they are no longer relevant.
       Css.remove(raw, 'width');
       Css.remove(cell, 'width');
+
+      const scope = Attribute.get(cell, 'scope');
+
+      if (Type.isString(scope)) {
+        Attribute.set(raw, 'scope', scope);
+      }
+
       return raw;
     };
   };
 
+  const merge = (cells: SugarElement[]) => {
+    const getScopeProperty = () => {
+      const baseScope = Attribute.get(cells[0], 'scope');
+      const scopes = [ 'row', 'col' ];
+
+      const isMixed = Arr.exists(cells.slice(1), (c) => {
```

Comment:
I don't believe this logic is going to work properly. What if you merge a `col` with a `rowgroup` or vice-versa a `row` with with a `colgroup`.

Reply:
Another thing that will cause issues here is if the `baseScope` is a group already. That'll cause it to incorrectly declare it as being mixed in some cases (eg expanding a `colgroup` by merging in another `col`).

Reply:
I did not think of that. Though it will only cause a problem if you merge `row` with `rowgroup`, as all other cases should result in `colgroup`, if I've not misunderstood the priority of col wrong. I'll see about fixing that case though.